elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -0,0 +1,412 @@
+import prairielearn as pl
+import lxml.html
+import random
+import chevron
+import base64
+import os
+import json
+import re
+
+# Read https://prairielearn.readthedocs.io/en/latest/devElements/
+# Official documentation on making custom PL
+
+
+def render_html_colour(score):
```

Comment:
Please us US spellings.
```suggestion
def render_html_color(score):
```

Reply:
Fixed in the latest commit